packages/playground/src/views/minting_view.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <view-layout>
+    <h2 class="mb-5 text-capitalize">Alpha software - all data subject to change</h2>
+    <v-form @change.prevent="mintingHash()" class="d-inline-flex w-100">
+      <FormValidator v-model="isValidForm">
+        <InputValidator
+          :value="receiptHash"
+          :rules="[
+            validators.required('Receipt hash is required.'),
+            validators.equal('Receipt hash must be 64 characters long.', 64),
+            validators.isHash('Invalid hash', 'sha256'),
+          ]"
+          valid-message="Receipt hash is valid."
+          #="{ props: validationProps }"
+          ref="textInput"
+        >
+          <VTextField
+            class="mr-5"
+            placeholder="Please insert your receipt hash"
+            v-model="receiptHash"
+            v-bind="{ ...validationProps }"
+            :loading="loading"
+        /></InputValidator>
+
+        <VBtn
+          type="submit"
+          color="primary"
+          variant="elevated"
+          :loading="loading"
+          :disabled="!isValidForm"
+          size="x-large"
+          rounded="lg"
+        >
+          Mint
```

Comment:
The page is just showing the minting details, so I think we shouldn't call it `Mint`, maybe `Show` or `View`.